packages/draw/src/renderables.ts

```diff
@@ -26,13 +26,15 @@ export class RenderableLink implements Drawable {
     );
   }
 
-  draw(canvas: Canvas, isActive: boolean): void {
+  draw(canvas: Canvas, isActive: boolean, isAnythingActive: boolean): void {
```

Comment:
This is probs dumb but what if we do `isActive?: boolean` with:

```
true = bright
undefined = normal
false = dimmed
```

Reply:
and remove isAnythingActive

Reply:
Would that change the logic at https://github.com/jollyjerr/mindgraph/pull/17/files#diff-85da206e66d53748eee13584274555243888de57e6a7d1e3e45e4a4f08e13cc7R126 to be something like:
```
if (activeDrawables.includes(d) || activeDrawables.length > 0) {
  d.draw(this, activeDrawables.includes(d))
} else {
  d.draw(this)
}
```
?

Reply:
That feels like there's some hidden logic that the end-user would have to be aware of. 

But I see where you're coming from, we're only really trying to encode 1 of 3 states in a single value. 

Maybe the second parameter to `draw` is just a `highlight` type with 3 possible values "dimmed" | "active" | "normal", or just an `enum``?

```
enum Highlight {
  Active
  Dimmed
  Normal
}
```

Reply:
Let's do that! I normally prefer string literals to enums (just simpler), but either way works

```ts
draw(canvas: Canvas, highlight: "active" | "dimmed" | "normal")
```

Reply:
Cool! I don't think I have a preference for one over the other so I'll try that out